### ISSUE TITLE
Realworld dataset request

### ISSUE THREAD

**Issue by chentuochao** [chentuochao/Sound_Bubble#2]
Hi, 
Our real-world dataset is available upon request. Currently,  the request form is not ready. Before the form is ready, you can reach out to soundbubble@cs.washington.edu to get the realworld data.

Currently, our real-world dataset are first open to research use.

**Comment by zoe199402** [comment]
Hello,

I have sent an email to request the dataset and look forward to your response.

Best regards

**Comment by chentuochao** [comment]
Hi, Currently our dataset only supports research use, please send the request via your institute/university/research department email

**Comment by zoe199402** [comment]
Yes, I applied using my university email. The email address is yuanxin@sues.edu.cn

**Comment by zoe199402** [comment]
Thank you for your response. I received the link, but it seems to be no longer working. Could you kindly provide an updated link? I would greatly appreciate it.
<img width="1143" alt="截屏2025-01-03 09 12 35" src="https://github.com/user-attachments/assets/596ef542-b44f-4889-a855-9afc57a3c80f" />

**Comment by chentuochao** [comment]
Hi I shared the data.
I think this issue can be closed
